core/src/main/java/com/datastax/oss/driver/internal/core/metadata/token/Murmur3Token.java

```diff
@@ -15,49 +15,14 @@
  */
 package com.datastax.oss.driver.internal.core.metadata.token;
 
-import com.datastax.oss.driver.api.core.metadata.token.Token;
-import com.datastax.oss.driver.shaded.guava.common.base.Preconditions;
-import com.datastax.oss.driver.shaded.guava.common.primitives.Longs;
-import edu.umd.cs.findbugs.annotations.NonNull;
 import net.jcip.annotations.Immutable;
 
 /** A token generated by {@code Murmur3Partitioner}. */
 @Immutable
-public class Murmur3Token implements Token {
-
-  private final long value;
+public class Murmur3Token extends TokenLong64 {
```

Comment:
This extraction should be in a separate commit for the sake of easier review.

Reply:
Separated into separate commit.